packages/react-native/ReactAndroid/src/main/java/com/facebook/hermes/instrumentation/HermesMemoryDumper.kt

```diff
@@ -4,15 +4,14 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  */
-
-package com.facebook.hermes.instrumentation;
+package com.facebook.hermes.instrumentation
 
 public interface HermesMemoryDumper {
-  boolean shouldSaveSnapshot();
+    public fun shouldSaveSnapshot(): Boolean
 
-  String getInternalStorage();
+    public fun getInternalStorage(): String
 
-  String getId();
+    public fun getId(): String
```

Comment:
I marked these as `String` and not `String?` since in Cxx the return type is `std::string` and not a pointer, but i do see some pointer shenanigans going on over there. Will leave it up to the person who reviews to give their thoughts as well. I can not find any use of this class anywhere, in Cxx or in the JVM land so i can't tell by usage